tuna/fin_class.py

```diff
@@ -211,21 +208,44 @@ def applicability(self):
 
     return True
 
-  def set_all_configs(self):
+  def set_all_configs(self, idx=0, num_blk=1):
     """Gathering all configs from Tuna DB to set up fin input file"""
     with DbSession() as session:
       query = session.query(
           self.dbt.config_table).filter(self.dbt.config_table.valid == 1)
 
       if self.label:
-        query_cfgs = session.query(self.dbt.job_table)\
-            .filter(self.dbt.job_table.reason == self.label)
+        query_cfgs = session.query(self.dbt.config_tags_table)\
+            .filter(self.dbt.config_tags_table.tag == self.label)
         rows = query_cfgs.all()
         ids = tuple([str(job_row.config) for job_row in rows])
         query = query.filter(self.dbt.config_table.id.in_(ids))
 
+      query = query.order_by(self.dbt.config_table.id.asc())
```

Comment:
Is asc() needed here?

Reply:
It doesn't matter which way actually. Is this a performance hit?

Reply:
it can be, so if not needed i would remove.